### PR TITLE
fix: getTxHistory [1]

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Motivation

It causes error when calling getTxHistory in the saga effect to fetch tx history on wallet-mobile.

### Acceptance Criteria
- Fix `getTxHistory` by passing network object to `getAddressFromPubkey`

### Security Checklist
- [x] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/new/wallet.js [hunk 1]
```diff
@@ -768,7 +768,7 @@ class HathorWallet extends EventEmitter {
         version: tx.version,
         ncId: tx.nc_id,
         ncMethod: tx.nc_method,
-        ncCaller: tx.nc_pubkey && getAddressFromPubkey(tx.nc_pubkey),
+        ncCaller: tx.nc_pubkey && getAddressFromPubkey(tx.nc_pubkey, this.getNetworkObject()),
```
Comment (pedroferreira1) [inline src/new/wallet.js:771]: We should have an integration test for this, since it was a bug before.

**Review by pedroferreira1 — CHANGES_REQUESTED** [review]

File: src/new/wallet.js (hunk 1 above)
Comment (r4mmer) [reply src/new/wallet.js:771]: Do we have a unit test at least?
If so, why did it not warn us of this?

**Review by r4mmer — APPROVED** [review]

File: src/new/wallet.js (hunk 1 above)
Comment (pedroferreira1) [reply src/new/wallet.js:771]: We can use the integration tests that we already have for nano contracts at https://github.com/HathorNetwork/hathor-wallet-lib/blob/master/__tests__/integration/nanocontracts/bet.test.js to call this method at least once to validate it. 

What do you think?

Comment (alexruzenhack) [reply src/new/wallet.js:771]: Done.

**Review by r4mmer — APPROVED** [review]

File: __tests__/integration/nanocontracts/bet.test.js [hunk 2]
```diff
@@ -301,6 +301,10 @@ describe('full cycle of bet nano contract', () => {
     for (const tx of ncHistory2.history) {
       expect(txIds).toContain(tx.hash);
     }
+
+    // Get tx history, it must be greater then zero
+    const txHistory = await wallet.getTxHistory();
+    expect(txHistory).not.toHaveLength(0);
```
Comment (pedroferreira1) [inline __tests__/integration/nanocontracts/bet.test.js:307]: Why don't you expect the correct number of transactions, which is 5?

Comment (alexruzenhack) [reply __tests__/integration/nanocontracts/bet.test.js:307]: I can update it, but the purpose was only to call `getTxHistory` with success, and this expect is sufficient for this purpose.

**Pull request merged**